server/controllers/applications/getMyApplication.js

```diff
@@ -1,3 +1,44 @@
-module.exports = (req, res) => {
-  res.send('get my appplication');
+const yup = require('yup');
+
+const getMyApplications = require('../../database/queries/applications/getMyApplications');
+const getMyApplication = require('../../database/queries/applications/getMyApplication');
```

Comment:
require (getMyApplications - getMyApplication) inside `server/database/queries/application/index.js
then you can require them here something like:
```js
const { getMyApplications, getMyApplications } = require('../../database/queries/applications');
```

Reply:
ok, thanks